src/test/browser/page-object/elements.ts

```diff
@@ -11,6 +11,7 @@ class Elements {
 
   protected rootElement?: webdriverio.Element;
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   [name: string]: any;
```

Comment:
Did you consider using `unknown` here?
That won't trigger the `any` rule.

Reply:
Yeah... That ends up giving me 82 new errors. I'm not worried about it since it's test code.